kubernetes/zenko/values.yaml

```diff
@@ -117,17 +117,42 @@ zenko-queue:
     enabled: true
   configurationOverrides:
     "offsets.topic.replication.factor": 3 # - replication factor for the offsets topic
-    "auto.create.topics.enable": true     # - enable auto creation of topic on the server
+    "auto.create.topics.enable": false    # - enable auto creation of topic on the server
     "min.insync.replicas": 2              # - min number of replicas that must acknowledge a write
-  partitionCount: *nodeCount
-  partitionFactor: *crrParallelism
+    "message.max.bytes": "5000000"        # - the largest record batch size allowed
   prometheus:
     jmx:
       enabled: true
     kafka:
       enabled: true
-zookeeper:
-  enabled: false # Disables zenko-queue from deploying its own quorum
+  topics:
+    - name: backbeat-gc
+      partitions: *crrParallelism
```

Comment:
If I'm not mistaken we're changing the number of partitions here, shall be reverted IMHO to the original value `partitionCount * partitionFactor`

Reply:
We were doing that because before we were setting partitions in a more global scope. But this gives you more freedom and the ability to do it per topic.

Reply:
Also, I think we should prevent the use of `Yaml` anchors and helm's templating facilities to have "factor" variables as much as we can. Both things are not a "thing" in the Helm community.